lib/index.js

```diff
@@ -115,6 +116,7 @@ function addRoutes(app) {
   app.options('/verifier/presentations', cors());
   app.post('/verifier/presentations', cors(), asyncHandler(async (req, res) => {
     let response;
+    const documentLoader = loader.build();
```

Comment:
See my other comment about the documentLoader arrangement here, but why is this being called here vs the top of the file?  Put another way, a fully configured `documentLoader` instance should be a global.

Reply:
+1, lets move it to top of the file.

Reply:
Moved 003a2a5